lib/msal-node/README.md

```diff
@@ -73,6 +74,19 @@ Before using `@azure/msal-node` you will need to register your app in the azure
 ```javascript
 npm install @azure/msal-node
 ```
+## Node Version Support
+MSAL Node will follow the [Long Term Support (LTS) schedule of the Node.js project](https://nodejs.org/en/about/releases/). Our support plan is as follows.
+
+Any major MSAL Node release:
+- Will support active LTS versions
+- Will drop support for any previously supported Node versions that have reached end of life
+- Will not support prerelease/preview/pending versions until they are stable
+
+| MSAL Node version | MSAL support status     | Supported Angular versions |
+|-------------------|-------------------------|----------------------------|
+| 1.x.x             | Active development      | 10(maintanence), 12, 14    |
```

Comment:
```suggestion
| 1.x.x             | Active development      | 10, 12, 14, 15    |
```

I don't think you need to label versions as maintenance, as the Node.js page details that.

Reply:
We do not want to list odd numbers, as per the norm??

Reply:
Ah okay, then we need to adjust the line above and the `engines` field.

Reply:
Suggestion updated

Reply:
What I think is this, we will support the range in `engines` since it is usually in between for any node updates, but specify we may not upgrade MSAL Node for odd numbered Node.js updates. I haven't made up my mind yet, we can discuss this and resolve offline.

Reply:
As discussed offline, updating to even versions for now. Let's wait on the feedback and update as needed.